api-samples/push/README.md

```diff
@@ -0,0 +1,23 @@
+# self.registration.pushManager.subscribe() userVisibleOnly
```

Comment:
Can we make this "Service worker with push notification" to be consistent with the manifest?